sdks/python/apache_beam/runners/interactive/sql/utils.py

```diff
@@ -161,7 +161,8 @@ class OptionsForm:
   generate PipelineOptions to run pipelines.
   """
   def __init__(self):
-    self.options = PipelineOptions()
+    # ignore all flags
+    self.options = PipelineOptions(flags={})
```

Comment:
Would this break any other use cases? How can we test for that?

Reply:
The current beam_sql magic tests passed. The magic does not use `flags`.

Reply:
BTW: I forgot to mention that I also tested the fix manually by running the beam_sql magic using our Dataflow Notebooks.

Reply:
To clarify: here `flags` is a parameter of the constructor, not the flags a Beam pipeline would use.
If not set as `{}`, the `PipelineOptions` is not initiated with a clean state and will use system's command line arguments (the current process uses, which can be anything unrelated to Beam). This is never useful in an interactive setup.